docs/admin/uninstall.md

```diff
@@ -14,18 +14,8 @@ nautobot-server migrate nautobot_chatops zero
 
 Remove the configuration you added in `nautobot_config.py` from `PLUGINS` & `PLUGINS_CONFIG`.
 
-## Post Upgrade
```

Comment:
I question this change. The migrations should be removed before uninstalling the app.

Reply:
Line 10 addresses this with `nautobot-server migrate nautobot_chatops zero`, I believe?

Reply:
Ah, I didn't look at the whole file! Thanks!